app/templates/main/emergencyContactInfo.html

```diff
@@ -0,0 +1,85 @@
+{% set title = "User Profile" %}
+{% extends "base.html" %}
+
+{% block scripts %}
+    {{super()}}
+    <script type="text/javascript" src="{{url_for('static', filename='js/emergencyContactInfo.js') }}?u={{lastStaticUpdate}}"></script>
+{% endblock %}
+
+{% block app_content %}
+
+<div class="text-center">
+    <h1> Emergency Contact </h1> <br>
+</div>
+
+<div id='username' hidden value='{{contactInfo.user}}'></div>
```

Comment:
if you're using this hidden tag the way i think you are, change this from a div to an input tag. if nothing else, it's better practice